guides/source/maintenance_policy.md

```diff
@@ -38,8 +38,9 @@ Bug Fixes
 ---------
 
 Only the latest release series will receive bug fixes. When enough bugs are
-fixed and its deemed worthy to release a new gem, this is the branch it happens
-from.
+fixed and its deemed worthy to release a new gem, it happens from the
+relevant x-y-stable branch. For example, a theoretical 7.0.1 bug fix release would
+be built from the 7-0-stable branch.
```

Comment:
I hope you don't mind, but I also took a stab at rewriting this.

```diff
diff --git a/guides/source/maintenance_policy.md b/guides/source/maintenance_policy.md
index 4dfc50654b..cb77a6bc35 100644
--- a/guides/source/maintenance_policy.md
+++ b/guides/source/maintenance_policy.md
@@ -37,9 +37,11 @@ in point releases.
 Bug Fixes
 ---------
 
-Only the latest release series will receive bug fixes. When enough bugs are
-fixed and its deemed worthy to release a new gem, this is the branch it happens
-from.
+Bug fixes are typically added to the main branch, and backported to the
+x-y-stable branch of the latest release series if there is sufficient need. When
+enough bugs fixes have been added to an x-y-stable branch, a new patch release
+is built from it. For example, a theoretical 1.2.2 patch release would be built
+from the 1-2-stable branch.
 
 In special situations, where someone from the Core Team agrees to support more series,
 they are included in the list of supported series.
```

I wanted to include the part about bug fixes typically being added to main and then backported, since I have seen confusion about that.  I also tried to mirror the previous section ("New Features") and the next section ("Security Issues") a bit, to tie them together.

Reply:
Thank you!

Reply:
I don't mind at all 😸